plugins/modules/database/saphana/hana_query.py

```diff
@@ -21,13 +21,17 @@
         type: str
         required: true
     user:
-        description: A dedicated username. Defaults to C(SYSTEM).
+        description: A dedicated username. The user could be also in hdbuserstore. Defaults to C(SYSTEM).
         type: str
         default: SYSTEM
+    userstore:
+        description: If true the user must be in hdbuserstore.
+        type: bool
+        default: false
```

Comment:
```suggestion
        default: false
        version_added: 3.5.0
```

Reply:
3.5.0 is scheduled for around August 10th (https://github.com/ansible-collections/community.general/issues/582). Seems fair, but maybe too short: the value is a suggestion, the field `version_added` is mandatory.